Dockerfile

```diff
@@ -13,6 +13,8 @@ RUN composer self-update --no-progress && \
 # Copy the working dir to the image's web root
 COPY . /var/www/html
 
+RUN chmod a+rwx /var/www/html/assets
```

Comment:
What is this for? The assets directory for the assetManager is created below with correct permissions.